tools/enso4igv/src/main/java/org/enso/tools/enso4igv/EnsoSbtClassPathProvider.java

```diff
@@ -180,13 +185,38 @@ private static EnsoSources[] computeSbtClassPath(EnsoSbtProject prj) {
                 }
             }
         }
-        return sources.toArray(new EnsoSources[0]);
+        if (prj.getProjectDirectory().getFileObject("src") instanceof FileObject src && src.isFolder()) {
+            for (var kind : src.getChildren()) {
+              TYPE: for (var type : kind.getChildren()) {
+                    for (var e : sources) {
+                      if (e.getRootFolder().equals(type)) {
+                        continue TYPE;
+                      }
+                    }
+
+                    var s = new OtherEnsoSources(kind.getNameExt(), type);
+                    sources.add(s);
+                }
+            }
+        }
+        return sources.toArray(new SourceGroup[0]);
+    }
+
+    private static FileObject findProjectFileObject(EnsoSbtProject prj, String path) {
+        if (path == null) {
+            return null;
+        }
+        if (path.startsWith("./")) {
```

Comment:
```suggestion
        if (!path.isAbsolute()) {
```
so that it works on Windows

Reply:
`path` is of type `String`, it doesn't have `isAbsolute` method in Java. Anyway using `/` is correct when working with `FileObject` (NetBeans abstraction over file). `FileObject` API is using `/` on all operating systems.